src/py42/sdk/queries/fileevents/filters/event_filter.py

```diff
@@ -1,12 +1,31 @@
 from py42.sdk.queries.fileevents.file_event_query import FileEventFilterStringField
+from py42.sdk.queries.query_filter import filter_attributes
 from py42.sdk.queries.query_filter import QueryFilterTimestampField
 
 
 class EventTimestamp(QueryFilterTimestampField):
-    """Class that filters events based on the timestamp of the event that occurred."""
+    """Class that filters events based on the timestamp of the event that occurred.
+
+    Example::
+        filter = EventTimestamp.within_the_period(EventTimestamp.SEVEN_DAYS)
```

Comment:
I like that you added support for `within_the_period`. Can we change it to be named `within_the_last`? Also, can the docstring list out the contants here and make it clear that they only work with `within_the_last`?

Reply:
Renamed. Added docstring.